io.github.yairm210.unciv.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.yairm210.unciv</id>
```

Comment:
```suggestion
  <id>io.github.yairm210.Unciv</id>
```

IMO it should be `io.github.yairm210.Unciv` and not `io.github.yairm210.unciv`.

Reply:
I do not feel strongly either way. I just left uncapitalized as it is easier to type, and domain names tend to be case-insensitive. If you feel it's better, I can make it that way.

Reply:
Usually the last part is the name of the program, which usually are capitalized. For example Steam: `com.valvesoftware.Steam`, Discord: `com.discordapp.Discord`, etc. It's not an issue, but I just wanted to point it out. Flatpak will automatically find UnCiv if you type `flatpak install unciv` (that's if it's available in a remote).